### DISCUSSION TITLE
no decode delegate for this image format `JFIF'

### DISCUSSION THREAD

**Discussion by betterapp** [ImageMagick/ImageMagick#7648] [Help]
### ImageMagick version

7.1.1-39

### Operating system, version and so on

Ubuntu

### Description

When I try to use identify
I get error:

identify: no decode delegate for this image format `JFIF' @ error/constitute.c/ReadImage/746.

What to do ?

**Comment by snibgo** [comment]
What was your command? What was your input image?

If the extension of your file is "JFIF" then that is the problem.

The command "`magick -list format`" will list the formats that your build of IM recognises.

**Comment by fmw42** [comment]
In Imagemagick, use "magick identify", not "identify".  All the tools use magick prefix, except for convert. Convert is changed to simply magick.

**Comment by betterapp** [comment]
I use linux "identify".

"The identify program is a member of the [imagemagick](https://linux.die.net/man/1/imagemagick)(1) suite of tools."
https://linux.die.net/man/1/identify

.JFIF is a allowed extension for files.
https://fileinfo.com/extension/jiff
https://en.wikipedia.org/wiki/JPEG_File_Interchange_Format

**Comment by snibgo** [comment]
What was your command? What was your input image?

**Reply by fmw42** [reply]
You are on Imagemagick 7, the commands have changed as I mentioned above. See https://imagemagick.org/script/porting.php#cli
